blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -41,7 +41,11 @@ const LinksBar = ({ customFields: { navigationConfig = {} } }) => {
 
   return (
     <>
-      <nav key={id} className="links-bar">
+      <nav
+        key={id}
+        className="links-bar"
+        aria-label={ariaLabel || 'More Links'}
```

Comment:
The `||` would never be called as it's the default set in the function arguments

Reply:
The default values in these components arent being passed for some reason. Im not exactly sure why but unless the defaults are placed here, the screen reader is not picking them up. looking into this